ext/MitsubaLoader/CElementShape.h

```diff
@@ -225,8 +225,8 @@ class CElementShape : public IElement
 		inline core::matrix3x4SIMD getAbsoluteTransform() const
 		{
 			auto local = transform.matrix.extractSub3x4();
-			if (type==CElementShape::INSTANCE && instance.parent)
-				return core::concatenateBFollowedByA(local,instance.parent->getAbsoluteTransform());
+			//if (type==CElementShape::INSTANCE && instance.parent)
+			//	return core::concatenateBFollowedByA(local,instance.parent->getAbsoluteTransform());
```

Comment:
what happened here!?

Reply:
instance.parent is always SHAPEHROUP by mitsuba docs which doesnt have tform so it makes no sense. I dont know how it worked, i did it in a way more understadnable for me

Reply:
https://en.wikipedia.org/wiki/Embrace,_extend,_and_extinguish

I made it possible to have an instance parent another one, transform-wise.